website/docs/addon_motionbuilder_admin.md

```diff
@@ -10,6 +10,27 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.MotionBuilder_Badge}
 </ReactMarkdown>
 
-:::note Work in progress
-This part of documentation is still work in progress.
+:::caution Unknown support for Motion Builder 2023 and below
+
+The AYON Motion Builder addon was initially developed with Motion Builder 2024 and 2025.
+Hence functionality was not tested in older versions than 2024 and may not work.
+:::
+
+## Supported Workflows
+
+The MotionBuilder addon currently enables you to import models, rigs, animations, and cameras. It also supports exporting animations and managing workfiles through saving, opening, and publishing.
+
+:::info Motion Builder Addon Settings
+The Motion Builder addon does not include individual settings, so it won't be listed in your studio's addon settings. However, this does not affect the addon's ability to utilize the core settings.
 :::
+
+## Publish Plugins
+
+The Motion Builder addon leverages the plugins from the core addon. Many of these plugins offer filters where you can specify motionbuilder as the host. For example:
+
+- **Validate Version:** Ensures that the workfile version is up to date with the latest in AYON. Access this validator at `ayon+settings://core/publish/ValidateVersion` and include Motion Builder's host name, which by default is `motionbuilder`.
+![](assets/motionbuilder/valid_version_plugin.png)
+
+
+- **Validate Outdated Container:** Checks that any loaded content is the most recent version. This validator is accessible at `ayon+settings://core/publish/ValidateOutdatedContainers`. Add Motion Builder's host name, which by default is `motionbuilder`.
+![](assets/motionbuilder/validate_outdated_containers.png)
```

Comment:
```suggestion
- **Validate Version:** Ensures that the workfile version is up to date with the latest in AYON. Access this validator at `ayon+settings://core/publish/ValidateVersion` and include MotionBuilder's host name, which is `motionbuilder`.
![](assets/motionbuilder/valid_version_plugin.png)


- **Validate Outdated Container:** Checks that any loaded content is the most recent version. This validator is accessible at `ayon+settings://core/publish/ValidateOutdatedContainers`. Add MotionBuilder's host name, which is `motionbuilder`.
![](assets/motionbuilder/validate_outdated_containers.png)
```

Use `MotionBuilder` instead of `MotionBuilder`
Also, as far as I know the host name should not be editable? so mentioning that `motionbuilder` is 'default' host name seems odd?

Reply:
> Also, as far as I know the host name should not be editable?

as far as I know, it's possible to change it in the application settings!
![image](https://github.com/user-attachments/assets/bc3a7baa-b65d-49fc-8f2d-8a9c6ccf759a)

Reply:
@iLLiCiTiT out of curiosity? Should it be editable?

And  you're correct - it's also still editable in `applications-1.0.0-dev.1`

Reply:
After working on https://github.com/ynput/ayon-documentation/pull/220 
I can tell that: 
This setting is editable for many reasons: 
1.  This name should be the host name hard coded in the addon. therefore, we need a way to allow specifying that value for applications addon.
2. It's possible for many applications to point to the same host name e.g. different nuke applications.

For motion builder app, applications addon will pick the Motion builder addon because the host name in settings matches [MotionBuilderAddon.host_name](https://github.com/ynput/ayon-motionbuilder/blob/68e2c76bf777540495fee5c4d33c8bbe95d06c66/client/ayon_motionbuilder/addon.py#L14). 

So, it's editable but is that user friendly or is it related to admins ? I can't really tell. 
and it's not related to this PR anyways.